src/Maestro/Maestro.Data/Models/DependencyFlow.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Maestro.Data.Models
+{
+    public class DependencyFlowEvent
```

Comment:
Is this class going to be "linked" to other classes in subsequent PRs? Right now it doesn't link to any other class in the model, right?

Reply:
As of my more recent change, it links to Branch. It also includes ChannelId, but isn't explicitly linked (I didn't want events to disappear just because a Channel was removed).